### PR TITLE
Chronos:  Move use-case to `chronos-tutorial` and modify page layout

### PR DESCRIPTION
**Pull request by liangs6212**

### CODE REVIEW

File: docs/readthedocs/source/doc/Chronos/QuickStart/index.md [hunk 1]
```diff
@@ -2,18 +2,97 @@
...
 
-    ![](../../../../image/colab_logo_32px.png)[Run in Google Colab](https://colab.research.google.com/github/intel-analytics/BigDL/blob/branch-2.0/python/chronos/colab-notebook/chronos_minn_traffic_anomaly_detector.ipynb) &nbsp;![](../../../../image/GitHub-Mark-32px.png)[View source on GitHub](https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/chronos/colab-notebook/chronos_minn_traffic_anomaly_detector.ipynb)
+    > ![](../../../../image/colab_logo_32px.png)[Run in Google Colab](https://colab.research.google.com/github/intel-analytics/BigDL/blob/branch-2.0/python/chronos/colab-notebook/chronos_minn_traffic_anomaly_detector.ipynb) &nbsp;![](../../../../image/GitHub-Mark-32px.png)[View source on GitHub](https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/chronos/colab-notebook/chronos_minn_traffic_anomaly_detector.ipynb)
+
+    In this guide we will demonstrate how to use _Chronos Anomaly Detector_ for real time traffic data from the Twin Cities Metro area in Minnesota anomaly detection.
+
+---------------------------
+
+- [**AutoTSEstimator with customized model**](../Overview/forecasting.html)
```
[the remark this answers is not in the document]
Comment (TheaperDeng) [reply docs/readthedocs/source/doc/Chronos/QuickStart/index.md:27]: Make this link to the notebook page. (same comment to other non-quickstart item)

File: docs/readthedocs/source/doc/Chronos/QuickStart/index.md [hunk 2]
```diff
@@ -2,18 +2,97 @@
...
+
+    > ![](../../../../image/GitHub-Mark-32px.png)[View source on GitHub](https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/chronos/use-case/network_traffic/network_traffic_autots_customized_model.ipynb)
+
+    In this notebook, we demonstrate a reference use case where we use the network traffic KPI(s) in the past to predict traffic KPI(s) in the future. We demonstrate how to use _AutoTSEstimator_ to adjust the parameters of a customized model.
+
+---------------------------
+
+- [**Network Traffic Forecasting with AutoTSEstimator**](../Overview/forecasting.html)
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/QuickStart/index.md:35]: Auto Tune the Prediction of Network Traffic at the Transit Link of WIDE

File: docs/readthedocs/source/doc/Chronos/QuickStart/index.md [hunk 3]
```diff
@@ -2,18 +2,97 @@
...
+
+    > ![](../../../../image/GitHub-Mark-32px.png)[View source on GitHub](https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/chronos/use-case/network_traffic/network_traffic_autots_forecasting.ipynb)
+
+    In this notebook, We demostrate how to use _AutoTS_ in project [Chronos][4] to do time series forecasting in an automated and distributed way.
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/QuickStart/index.md:39]: In this notebook, we demostrate a reference use case where we use the network traffic KPI(s) in the past to predict traffic KPI(s) in the future. We demostrate how to use AutoTS in project Chronos to do time series forecasting in an automated and distributed way.

File: docs/readthedocs/source/doc/Chronos/QuickStart/index.md [hunk 4]
```diff
@@ -2,18 +2,97 @@
...
+
+---------------------------
+
+- [**Univariate and Multivariate Forecasting**](../Overview/forecasting.html#time-series-forecasting-concepts)
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/QuickStart/index.md:43]: Multivariate Forecasting of Network Traffic at the Transit Link of WIDE

File: docs/readthedocs/source/doc/Chronos/QuickStart/index.md [hunk 5]
```diff
@@ -2,18 +2,97 @@
...
+
+    > ![](../../../../image/GitHub-Mark-32px.png)[View source on GitHub](https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/chronos/use-case/network_traffic/network_traffic_model_forecasting.ipynb)
+
+    In this notebook, We demostrate how to do univariate forecasting (predict only 1 series), and multivariate forecasting (predicts more than 1 series at the same time) using Project Chronos.
+
+---------------------------
+
+- [**Multivariate Multistep Forecasting**](../Overview/forecasting.html#time-series-forecasting-concepts)
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/QuickStart/index.md:51]: Multistep Forecasting of Network Traffic at the Transit Link of WIDE

File: docs/readthedocs/source/doc/Chronos/QuickStart/index.md [hunk 6]
```diff
@@ -2,18 +2,97 @@
...
+
+    In this notebook, We demostrate how to use _AutoTS_ in project [Chronos][4] to do time series forecasting in an automated and distributed way.
+
+---------------------------
+
+- [**Univariate and Multivariate Forecasting**](../Overview/forecasting.html#time-series-forecasting-concepts)
+
+    > ![](../../../../image/GitHub-Mark-32px.png)[View source on GitHub](https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/chronos/use-case/network_traffic/network_traffic_model_forecasting.ipynb)
+
+    In this notebook, We demostrate how to do univariate forecasting (predict only 1 series), and multivariate forecasting (predicts more than 1 series at the same time) using Project Chronos.
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/QuickStart/index.md:47]: In this notebook, we demonstrate a reference use case where we use the network traffic KPI(s) in the past to predict traffic KPI(s) in the future. We demostrate how to do univariate forecasting (predict only 1 series), and multivariate forecasting (predicts more than 1 series at the same time) using Project Chronos.

File: docs/readthedocs/source/doc/Chronos/QuickStart/index.md [hunk 7]
```diff
@@ -2,18 +2,97 @@
...
+
+---------------------------
+
+- [**Multivariate Multistep Forecasting**](../Overview/forecasting.html#time-series-forecasting-concepts)
+
+    > ![](../../../../image/GitHub-Mark-32px.png)[View source on GitHub](https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/chronos/use-case/network_traffic/network_traffic_multivariate_multistep_tcnforecaster.ipynb)
+
+    In this notebook, We demostrate how to do multivariate multistep forecasting using Project Chronos.
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/QuickStart/index.md:55]: In this notebook, we demonstrate a reference use case where we use the network traffic KPI(s) in the past to predict traffic KPI(s) in the future. We demostrate how to do multivariate multistep forecasting using Project Chronos.

File: docs/readthedocs/source/doc/Chronos/QuickStart/index.md [hunk 8]
```diff
@@ -2,18 +2,97 @@
...
+
+---------------------------
+
+- [**Stock Price Prediction**](../Overview/forecasting.html)
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/QuickStart/index.md:59]: Stock Price Prediction with LSTMForecaster

File: docs/readthedocs/source/doc/Chronos/QuickStart/index.md [hunk 9]
```diff
@@ -2,18 +2,97 @@
...
+
+    > ![](../../../../image/GitHub-Mark-32px.png)[View source on GitHub](https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/chronos/use-case/fsi/stock_prediction_prophet.ipynb)
+
+    In this notebook, we demonstrate a reference use case where we use historical stock price data to predict the future price using the ProphetForecaster and AutoProphet. The dataset we use is the daily stock price of S&P500 stocks during 2013-2018 [data source](https://www.kaggle.com/camnugent/sandp500/).
+
+    Reference: *<https://facebook.github.io/prophet>*, *<https://github.com/jwkanggist/tf-keras-stock-pred>*
+
+---------------------------
+
+- [**Unsupervised Anomaly Detection**](../Overview/anomaly_detection.html)
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/QuickStart/index.md:79]: Unsupervised Anomaly Detection for CPU Usage

File: docs/readthedocs/source/doc/Chronos/QuickStart/index.md [hunk 10]
```diff
@@ -2,18 +2,97 @@
...
+
+    > ![](../../../../image/GitHub-Mark-32px.png)[View source on GitHub](https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/chronos/use-case/AIOps/AIOps_anomaly_detect_unsupervised.ipynb)
+
+    We demonstrates how to perform anomaly detection based on Chronos's built-in [DBScanDetector][DBScan], [AEDetector][AE] and [ThresholdDetector][Threshold].
+
+---------------------------
+
+- [**Unsupervised Anomaly Detection based on Forecasts**](../Overview/anomaly_detection.html)
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/QuickStart/index.md:87]: Anomaly Detection for CPU Usage Based on Forecasters

**Comment by liangs6212** [comment]
https://smalldl.readthedocs.io/en/chronos-tutorial/doc/Chronos/QuickStart/index.html

**Review by TheaperDeng — APPROVED** [review]
LGTM

**Comment by liangs6212** [comment]
http://10.112.231.51:18888/view/BigDL-2.0-PR/job/BigDL-Chronos-PR-Validation/233/
http://10.112.231.51:18888/job/BigDL-PRVN-chronos-Python-Spark-2.4-py36-ray-part1/271/

**Pull request merged**